homeassistant/components/profiler/__init__.py

```diff
@@ -143,6 +180,22 @@ def _dump_log_objects(call: ServiceCall):
         schema=vol.Schema({vol.Required(CONF_TYPE): str}),
     )
 
+    async_register_admin_service(
+        hass,
+        DOMAIN,
+        SERVICE_LOG_THREAD_FRAMES,
+        _async_dump_thread_frames,
+        schema=vol.Schema({}),
```

Comment:
Schema is optional.

Reply:
Cleaned up the existing unneeded one as well